tests/Kernel/Paragraphs/ContentRowTest.php

```diff
@@ -152,9 +151,6 @@ public function testRendering(): void {
    * Tests that side menu title translations are rendered correctly.
    */
   public function testSideMenuTranslation(): void {
-    // Create "French" language.
-    ConfigurableLanguage::createFromLangcode('fr')->save();
```

Comment:
Why did you remove? Where is this created?

Reply:
The languages are created in ParagraphsTestBase.